lib/portlayer/attach/connector.go

```diff
@@ -167,7 +167,7 @@ func (c *Connector) processIncoming(conn net.Conn) {
 		// the tether side (in tether_linux.go) or alignment may not happen.
 		// The PL sends the first SYN in the handshake and if the tether is not
 		// waiting, the handshake may never succeed.
-		ctx, cancel := context.WithTimeout(context.TODO(), 50*time.Millisecond)
+		ctx, cancel := context.WithTimeout(context.TODO(), time.Second)
```

Comment:
Can you update the comment? Looks like the ticker has been moved to `cmd/tether/attach.go`.  Also why is the timeout being made larger?